sdk-setup/src/sdk-manage

```diff
@@ -509,6 +509,18 @@ object_zypper() {
     local object=$1 args=("${@:2}")
 
     enter "$object" zypper --non-interactive "${args[@]}"
+    case $? in
+        100|101|102)
+            # Ignored exit codes:
+            # 100 - ZYPPER_EXIT_INF_UPDATE_NEEDED
+            # 101 - ZYPPER_EXIT_INF_SEC_UPDATE_NEEDED
+            # 102 - ZYPPER_EXIT_INF_REBOOT_NEEDED
+            :
```

Comment:
I would consider an explicit `return 0` as a more readable alternative :)

Reply:
Can change that however you prefer it. I agree the meaning of : isn't that obvious.